packages/dds/cell/src/cell.ts

```diff
@@ -43,6 +43,42 @@ interface ICellValue {
     value: unknown;
 }
 
+// TODO: this should reference a shared interface in @fluidframework/runtime-definitions so it's usable from
+// here and @fluidframework/attributor
+interface AttributionInfo {
```

Comment:
Rather than store this here, we should be storing a single `AttributionKey` on the SharedCell. It's up to the user to translate that attribution key into user/timestamp information if they so choose.